__tests__/app.test.js

```diff
@@ -111,3 +111,25 @@ describe("/api/articles", () => {
   })
   
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("get all comments for a specified article", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body;
+      expect(comments).toHaveLength(11);
+      comments.forEach((comment) => {
+        expect(comment.article_id).toBe(1);
```

Comment:
I would expect this test to also show that each comment has the following properties:
comment_id
votes
created_at
author
body